.github/workflows/go-build.yaml

```
@@ -15,6 +30,10 @@ on:
         description: 'Additional build flags to be passed.'
         required: true
         type: string
+      GO_CGO_ENABLED:
+        description: 'Whether to enable CGO. Set to 1 to enable CGO. If not provided, the default is 0.'
+        default: '0'
```

actionlint (1.7.4)

input of workflow_call event "GO_CGO_ENABLED" is typed as boolean. its default value must be true or false but got "0"

(events)